README.md

```diff
@@ -275,6 +275,23 @@ We are trying to draw a timetable using LazyLayout, a base implementation of Laz
 
 https://github.com/DroidKaigi/conference-app-2022/blob/91715b461b3162eb04ac58b79ba39ccdf21cf222/feature-sessions/src/main/java/io/github/droidkaigi/confsched2022/feature/sessions/Timetable.kt#L73
 
+## Jetpack Compose preview in GitHub comments
+
+If we can check Compose's Preview during GitHub code review, we can efficiently review changes.  
```

Comment:
A couple suggestion, I'm not native as well, but these can be slightly better (hopefully):

```suggestion
If we can review the Compose's Preview during GitHub code review, we can address the issue effectively.  
```